android/beagle/src/main/java/br/com/zup/beagle/android/components/Image.kt

```diff
@@ -100,19 +103,28 @@ data class Image(
         }
     }
 
-    private fun ImageView.loadImage(
-        path: ImagePath.Remote) {
-        Glide.with(this)
-            .load(path.url.formatUrl())
-            .into(object : CustomTarget<Drawable>() {
-                override fun onResourceReady(resource: Drawable, transition: Transition<in Drawable>?) {
-                    this@loadImage.setImageDrawable(resource)
-                    this@loadImage.requestLayout()
-                }
+    private fun ImageView.loadImage(path: ImagePath.Remote) {
+        CoroutineScope(CoroutineDispatchers.IO).launch {
```

Comment:
I'm not sure, but I think that the scope of this coroutine is global. What do you think of assigning this coroutine to the `lifecycleowner` so that when the screen is no longer visible, the request will be automatically canceled?
```suggestion
    private fun ImageView.loadImage(path: ImagePath.Remote, rootView: RootView) {
        rootView.getLifecycleOwner().lifecycleScope.launch {
```

Reply:
Done!